packages/block-editor/src/components/inserter/menu.js

```diff
@@ -341,4 +341,16 @@ function InserterMenu(
 	);
 }
 
-export default forwardRef( InserterMenu );
+export const PrivateInserterMenu = forwardRef( InserterMenu );
+
+function PublicInserterMenu( props, ref ) {
```

Comment:
As far as I've seen the inserter menu component is:

- used exclusively in the block editor package
- used exclusively in the inserter component in the block editor package
- not specifically exported anywhere

What do we need the private component for?